### DISCUSSION TITLE
[Auth0 Client] Access token and refresh

### DISCUSSION THREAD

**Discussion by georgev-97** [mia-platform/community#381] [Q&A]
### Which section?

Console

### Which is the version?

12

### Description

Good morning everybody,

since yesterday night my auth0-client is throwing this error:
`{"error":"access token in redis is empty","level":50,"msg":"failed token retrieval: access token in redis is empty","reqId":"a29dbcfa-7bdc-4d67-9254-e227534fbfa5","time":1706683944657}`

Shouldn't it request automatically another token from the management api if it's not present on redis?

**Comment by malta895** [comment]
Hi @georgev-97,
can you check that the env `SESSION_DURATION_SECONDS` matches the token expirations set on [Auth0 settings](https://auth0.com/docs/secure/tokens/access-tokens/update-access-token-lifetime)?

**Reply by georgev-97** [reply]
Yes, it's 86400 on both

**Reply by malta895** [reply]
From which auth0-client API does that log come from?

**Reply by georgev-97** [reply]
POST /users

**Reply by georgev-97** [reply]
I fixed by restarting the auth0-client

**Reply by malta895** [reply]
Understood, thanks! In the meantime we are investigating the issue and see if we can do something to improve the behavior in this situation.
